tests/RestfulRenderCacheTestCase.test

```diff
@@ -245,7 +247,7 @@ class RestfulRenderCacheTestCase extends \RestfulCurlBaseTestCase {
     // Try to access the cached resource.
     $response = $this->httpRequest($path);
     // The user should get a 403.
-    $this->assertEqual($response['code'], 403, 'Access denied for anonymous user.');
+    $this->assertEqual($response['code'], 401, 'Access denied for anonymous user.');
```

Comment:
Yikes. I'll fix the comment.

401 means authorization error, meaning that the requester meant to be identified with a Drupal user but the credentials were bad.

403 is more for, _I know who you are, and you cannot do that_.